### ISSUE TITLE
v1.19.2 does not start

### ISSUE THREAD

**Issue by Chicken** [httptoolkit/httptoolkit-desktop#79]
AppImage on archlinux, v1.19.1 works fine. v1.19.2 complains about tslib missing
```
$ ./HttpToolkit-1.19.2.AppImage
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'tslib'
Require stack:
- /tmp/.mount_HttpToZI3zeP/resources/app.asar/build/index.js
-
    at Module._resolveFilename (node:internal/modules/cjs/loader:1232:15)
    at s._resolveFilename (node:electron/js2c/browser_init:2:120689)
    at Module._load (node:internal/modules/cjs/loader:1058:27)
    at c._load (node:electron/js2c/node_init:2:16955)
    at Module.require (node:internal/modules/cjs/loader:1318:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/tmp/.mount_HttpToZI3zeP/resources/app.asar/build/index.js:3:17)
    at Module._compile (node:internal/modules/cjs/loader:1484:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1564:10)
    at Module.load (node:internal/modules/cjs/loader:1295:32)
Trace/breakpoint trap (core dumped)
```

**Comment by pimterry** [comment]
Hmm, that's interesting. This was literally just released, this is the first time appimage downloads have been officially enabled. Does the zip version launch for you or is that broken too?

**Comment by pimterry** [comment]
You can also test the v1.19.1 appimage here: https://github.com/httptoolkit/httptoolkit-desktop/releases/tag/v1.19.1

**Comment by Chicken** [comment]
v1.19.1 AppImage and zip both work, v1.19.2 AppImage and zip neither work

**Comment by pimterry** [comment]
Hmm, ok. I've rolled the website back to offer v1.19.1 for the moment, I'll look into v1.19.2 and see what's up there now.

**Comment by Chicken** [comment]
Broke in the commit "Update Electron to latest v31" (249f749ad2c362e438733e17fc650d34be543ea0)

**Comment by Chicken** [comment]
The aforementioned commit also updated the lockfile and for some reason marked tslib as dev (more like, why wasn't it marked as dev before, it so in the package.json). Simply moving tslib in package.json from devDependencies to dependencies fixes the issue.

**Comment by pimterry** [comment]
Thanks @Chicken, that is indeed it. Looks like we only ever had tslib as a devdep even though it was actually required at runtime, and it was only being included at runtime because it was a runtime dependency of the sentry package here (that was the only reason this worked before). 

When the sentry package was updated there as part of that dep update, their tslib update created a conflict in version numbers, which meant it was deduplicated from the top-level dep (so the top-level dep was now dev-only), which meant we suddenly had no runtime tslib available at all.

Whew, complicated. Fix here: https://github.com/httptoolkit/httptoolkit-desktop/commit/bddd09f48fe260d64021b8b2fb32d516cbbc1cbb.

@Chicken can you try downloading `linux-distributables` from https://github.com/httptoolkit/httptoolkit-desktop/actions/runs/12470209907 and test it works for you now? Seems OK in my testing but good to be sure. If that's all working, I'll bump it to 1.19.3 and re-release in just a minute.

**Comment by Chicken** [comment]
For some reason GitHub is only letting me download it at like 500kbps so that would take forever :joy: Locally built one works fine though. Seems good to release.

**Comment by pimterry** [comment]
No worries. Seems to work for me locally at least now and the fix makes sense. I've bumped & released v1.19.3 :+1:. Thanks for reporting this! Let me know if you run into any other issues at all.
